samples/wrapper/wrapper.cpp

```diff
@@ -36,15 +36,13 @@
 //
 // Contributor: Andrew J. Younge (ajy4490@umiacs.umd.edu)
 
-// comment out the following to disable checking that
-// executables are signed.
-// Doing so introduces a security vulnerability.
-//
-#define CHECK_EXECUTABLES
-
-#ifndef _WIN32
-#include "config.h"
+#ifdef CMAKE
```

Comment:
If you use already existing functions' implementations, this complete block is not needed